e2e_tests/tests/per_provider/normal_tests/aead.rs

```diff
@@ -213,3 +258,502 @@ fn aead_encrypt_ccm_encrypt_decrypt() {
 
     assert_eq!(&PLAINTEXT[..], plaintext.as_slice());
 }
+
+#[cfg(feature = "cryptoauthlib-provider")]
```

Comment:
The other tests in this file do not have that `cfg` gate. It would be nice to remove to make sure those tests also pass on the Mbed Crypto provider (our reference provider implementation). Do you know if that's possible? Feel free to just remove it to see if the CI pass :)

Reply:
Done.